decred_wallet/Features/Security/PinInputView.swift

```diff
@@ -37,17 +38,20 @@ class PinInputView: UIView {
     }
     
     override func draw(_ frame: CGRect) {
-        self.drawCells(in: frame)
-    }
-    
-    func drawCells(in frame: CGRect) {
+        
```

Comment:
This empty line is not necessary.

Reply:
Fixed in commit e4174d51aed804a470b8d42f08229756f8993915